Makefile

```diff
@@ -242,7 +242,7 @@ clear-deployment:
 	-python3 ./tools/clear_deployment.py --delete-namespace $(APPLY_NAMESPACE) --delete-pvc $(DELETE_PVC) --namespace "$(NAMESPACE)" --profile "$(PROFILE)" --target "$(TARGET)" || true
 
 clean-onprem:
-	podman pod rm -f -i assisted-installer
+	podman pod rm -f assisted-installer || true
```

Comment:
Why this change?

Reply:
we can't run interactively on jenkins

Reply:
That flag is not for interactive mode. `-i, --ignore                    Ignore errors when a specified pod is missing`

This change goes from ignoring just the error when the pod is missing, to instead ignoring all errors. While a minor issue, I think it's a regression.